Sl deprecation (#4620)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/74-Dec-2023/legacy-sl.md

This deprecation means dbt Metrics and the legacy Semantic Layer will no longer be supported. We will also remove the product from the dbt Cloud user interface and documentation site.

### Why this change?

We understand that changes of this nature can be disruptive and we believe in the potential of the new direction. The [re-released dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), powered by MetricFlow, is a significantly improved foundation that enables more flexible query generation, faster performance, and a more dynamic consumption experience. It’s a step towards a brighter future for dbt and its community.

### Key changes

- Legacy Semantic Layer and dbt Metrics will officially be deprecated on December 15th, 2023.
- MetricFlow has replaced dbt Metrics for the construction of semantic logic. The `dbt_metrics` package will no longer be supported.
- Exports replaces Materializations. Exports will be available in December or January in dbt Cloud and replaces the previous materializations functionality.

> 

Author: rather than materialization, should we say `metrics_calulate` so users dont confuse it with the dbt materializations?

Author: @rpourzand gave feedback and updating to reflect that

---
title: "Deprecation: dbt Metrics and the legacy dbt Semantic Layer is now deprecated"
description: "December 2023: For users on dbt v1.5 and lower, dbt Metrics and the legacy dbt Semantic Layer has been deprecated. Use the migration guide to migrate to and access the latest dbt Semantic Layer. "
sidebar_label: "Deprecation: dbt Metrics and Legacy dbt Semantic Layer"
sidebar_position: 09
date: 2023-12-15
---

> dbt Labs has deprecated dbt Metrics and the legacy dbt Semantic Layer, both supported on dbt version 1.5 or lower. These changes will be in effect on _December 15th, 2023_.

Reviewer: Same feedback here as below

> dbt Labs has deprecated dbt Metrics and the legacy dbt Semantic Layer, both supported on dbt version 1.5 or lower. These changes will be in effect on _December 15th, 2023_.

Author: ```suggestion
dbt Labs has deprecated dbt Metrics and the legacy dbt Semantic Layer, both supported on dbt version 1.5 or lower. This change starts on December 15th, 2023.
```

> This deprecation means dbt Metrics and the legacy Semantic Layer will no longer be supported. We will also remove the feature from the dbt Cloud user interface and documentation site.

Reviewer: Is this release note intended to go out before or on the 15th? May want to consider changing the tense in this doc if it's going on the 15th to drive home that it's gone now
```suggestion
This deprecation means dbt Metrics and the legacy Semantic Layer are no longer supported. We also removed the feature from the dbt Cloud user interface and documentation site.
```

### Why this change?

The [re-released dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), powered by MetricFlow, offers enhanced flexibility, performance, and user experience, marking a significant advancement for the dbt community.

### Key changes and impact

- **Deprecation date** &mdash; The legacy Semantic Layer and dbt Metrics will be officially deprecated on December 15th, 2023.
- **Replacement** &mdash; [MetricFlow](/docs/build/build-metrics-intro) replaces dbt Metrics for defining semantic logic. The `dbt_metrics` package will no longer be supported post-deprecation.

> - **New feature** &mdash; Exports replaces the materializations functionality and will be available in dbt Cloud in December or January.

Author: ```suggestion
- **New feature** &mdash; Exports replaces the materializing data with `metrics.calculate` functionality and will be available in dbt Cloud in December or January.
```
